content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md

```diff
@@ -0,0 +1,133 @@
+---
+weight: 40
+title: Release 10.16.0.0
+layout: bundle
+---
+
+### Improvements in Digital Twin Manager
+
+<div><table ><colgroup>
+<col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>The DTM application is integrated with Gainsight to capture the data.</td>
+<td>CTM-130</td>
+</tr>
+
+<tr>
+<td>Added a warning notification if a custom property associated with an asset is deleted using Cumulocity REST APIs when an asset is viewed.</td>
+<td>CTM-141</td>
+</tr>
+
+<tr>
+<td>A confirmation pop-up is shown in the <b>Property library</b> page for every delete operation.</td>
+<td>CTM-159</td>
+</tr>
+
+<tr>
+<td>Implemented a maximum file size field for custom properties of type file.</td>
+<td>CTM-175</td>
+</tr>
+
+<tr>
+<td>Added a confirmation popup which states that the changes made to the asset type will not be reflected in the assets that are currently associated with it.</td>
+<td>CTM-144</td>
+</tr>
+
+<tr>
+<td>Added a confirmation popup which states that the changes made to the custom property will not be reflected in the assets that are currently associated with it's asset type.</td>
+<td>CTM-147</td>
+</tr>
+
+<tr>
+<td>An alert save pop-up is displayed in the localization page.</td>
+<td>CTM-155</td>
+</tr>
+
+</tbody></table></div>
+
+### Fixes in Digital Twin Manager
+
+<div><table ><colgroup>
+<col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>The <b>Import assets</b> button is not visible for the last asset in the hierarchy now.</td>
+<td>CTM-170</td>
+</tr>
+
+<tr>
+<td>The loading message in the localization page is corrected.</td>
+<td>CTM-154</td>
+</tr>
+
+<tr>
+<td>An issue observed during hierarchy creation is fixed.</td>
+<td>CTM-125</td>
+</tr>
+
+<tr>
+<td>An issue while creating subassets when there are both required and optional child asset types at same level is fixed.</td>
+<td>CTM-167</td>
+</tr>
+
+<tr>
+<td>Changes made in the asset type page are getting saved now.</td>
+<td>CTM-157</td>
+</tr>
+
+<tr>
+<td>Provided a fix for the Add child asset window which became unresponsive for assets that had a custom property deleted from the backend.</td>
```

Comment:
```suggestion
<td>A fix has been fix for the <b>Add child asset</b> window which became unresponsive for assets that had a custom property deleted from the backend.</td>
```

Reply:
I think you mean"A fix has been provided" not "A fix has been fix"